clustered/build.gradle

```diff
@@ -15,14 +15,13 @@
  */
 
 ext {
-    terracottaPlatformVersion = '5.0.0.beta'
-    terracottaApisVersion = '1.0.0.beta'
+    terracottaPlatformVersion = '5.0.3.beta'
+    terracottaApisVersion = '1.0.3.beta'
     terracottaCoreVersion = '5.0.0-beta2'
-    coordinatorVersion = terracottaPlatformVersion
     clusteredMapVersion = terracottaPlatformVersion
     offheapResourceVersion = terracottaPlatformVersion
     entityApiVersion = terracottaApisVersion
-    entityTestLibVersion = terracottaApisVersion
+    entityTestLibVersion = '1.0.3.beta'
```

Comment:
I meant the rerouting of how versions are defined.  I think I understand now that this is make it so that each 'repository' in which the source for these projects is defined has an equivalent property in the build file here.  I think that makes sense but the naming is a bit off (`entityTestLibVersion` instead of `passthroughTestingVersion`), and there is no convention on whether each module should have it's own version variable that is defined to be equal to the "repo version variable" (the things in platform for example) or if there should only be variables for the 'repo', and the module should use that one directly (entityTestLibVersion).  Would be nice to make all this consistent if we're going to change stuff.